tests/test_project.py

```diff
@@ -23,6 +24,7 @@
 from test_job import TestJobBase
 
 import signac
+from signac.__main__ import SHELL_HISTORY_FN
```

Comment:
In general, we should not import from `__main__` modules. See previous comment about centralizing this definition.

Reply:
See my reply. I don't think moving the definition just for the sake of the test makes sense; if it comes down to that, perhaps we really should be hardcoding the values into the test instead.

Reply:
Based on your argument on the other thread, I’m fine with hardcoding it.